zoho_mcp/resources.py

```
@@ -7,18 +7,14 @@
 
 import logging
 from datetime import datetime, timedelta
+from typing import List
```

Ruff (0.12.2)

`typing.List` imported but unused

Remove unused import: `typing.List`

(F401)